### PR TITLE
Developed create student route and update student route.

### PR DESCRIPTION
**Pull request by yashodgayashan**
## Important

* The update route must contain query parameter called batch

### CODE REVIEW

File: BackendNew/controllers/students.js [hunk 1]
```diff
@@ -214,6 +214,26 @@ const BasicStudent = function(student) {
...
+  this.Sem1GPA = student.gpa.first_sem;
+  this.Sem2GPA = student.gpa.second_sem;
+  this.Sem3GPA = student.gpa.third_sem;
+  this.Sem4GPA = student.gpa.fourth_sem;
+  this.SGPA = student.gpa.SGPA;
+  this.PreferedArea1 = student.PreferedArea1;
+  this.PreferedArea2 = student.PreferedArea2;
+  this.PreferedArea3 = student.PreferedArea3;
+};
+
```
Comment (hasinichathu) [inline BackendNew/controllers/students.js:314]: Isn't it good to have separate folder for models?

File: BackendNew/controllers/students.js [hunk 2]
```diff
@@ -204,6 +204,45 @@ exports.updateStudent = (req, res) => {
...
 };
 
+const updateStudentInfo = (req, isUpdated) => {
+  var studentId = req.params.studentId;
+  students.getStudentCreater(studentId, (err, result) => {
+    if (err) {
+      isUpdated(err, null);
+    } else {
+      // Get createBy user.
+      var createdBy = result.creater;
```
Comment (hasinichathu) [inline BackendNew/controllers/students.js:214]: What's the use of getting creator here?

Comment (yashodgayashan) [reply BackendNew/controllers/students.js:214]: Have remove it. Forgot it.

File: BackendNew/controllers/students.js (hunk 1 above)
Comment (yashodgayashan) [reply BackendNew/controllers/students.js:314]: Yeah. Will move.

Comment (yashodgayashan) [reply BackendNew/controllers/students.js:314]: @hasinichathu there is a seperate folder for models? then what should we do?

File: BackendNew/controllers/students.js (hunk 2 above)
Comment (yashodgayashan) [reply BackendNew/controllers/students.js]: @hasinichathu Resolved it.

File: BackendNew/controllers/students.js (hunk 1 above)
Comment (hasinichathu) [reply BackendNew/controllers/students.js:296]: That means?? We can simply import the model when necessary. I saw several places where same model is used

Comment (yashodgayashan) [reply BackendNew/controllers/students.js:296]: Yeah. Will make a separate file for models

**Comment by yashodgayashan** [comment]
@hasinichathu Resolved requested changes.

**Pull request merged**